deploy/catalog_resources/community/7.8/kiecloud-operator.7.8.0.clusterserviceversion.yaml

```diff
@@ -104,6 +104,12 @@ spec:
         path: consoleHost
         x-descriptors:
         - urn:alm:descriptor:org.w3:link
+      - description: The reference to the generated NamespacedDashboard ConsoleLink
```

Comment:
I regenerated CSV because I forgot to add ConsoleLink to the status

Reply:
Also waiting to understand why ConsoleLink is needed in CR Status

Reply:
Thinking you need something stable that won't change or go away. The k8s resource ID of BC should give you that.

Reply:
Do you mean the KieApp UID? I think I could use that instead, yes.